examples/.github/workflows/push.yml

```diff
@@ -0,0 +1,12 @@
+on: push
+name: Main
+jobs:
+  pHP-CS-Fixer:
```

Comment:
```suggestion
  PHP-CS-Fixer:
```

Reply:
Same as the other one

>The github migration tools was going for pHP-CS-Fixer: so I figured that key should be lower case at least for the first character, hence the lowercase.
>Do you think we can leave it as is (lower)?

I updated it to be all lowercase for now, let me know.